Blink_ActiveWorkGoesHere/blink.js

```diff
@@ -0,0 +1,226 @@
+/*
+ * Blink object. This class handles loading the necessary assets as well as defines
+ * the update and draw function.
+ *
+ * Single constructor takes in the game context as its parameter. (There is no default) 
+ * 
+ */
+class Blink
+{
+    /**
+     * Single constructor for blink. Loads assets and sets intial parameters including
+     * the speed, starting x/y position, jump height, etc.
+     * 
+     * @constructor
+     * @param {any} game A reference to the game engine.
+     */
+    constructor(game)
+    {
+        // Load all assets for character first. There's quite a few.
+
+        this.standLeftAnimation = new Animation
```

Comment:
Added in all the animations for face left/right, slash left/right and jump left/right. Not all are programmed yet though.